service/matching/backlog_manager_test.go

```diff
@@ -308,3 +309,89 @@ func newBacklogMgr(controller *gomock.Controller) *backlogManagerImpl {
 func defaultContextInfoProvider(ctx context.Context) context.Context {
 	return ctx
 }
+
+func TestBacklogManager_AddingTasksIncreasesBacklogCounter(t *testing.T) {
```

Comment:
These are not new tests but have been moved from `ack_manager_test.go` and have been renamed to be consistent.